src/stack-hci/azext_stack_hci/aaz/latest/stack_hci/cluster/_create.py

```diff
@@ -56,6 +56,11 @@ def _build_arguments_schema(cls, *args, **kwargs):
         # define Arg Group "Cluster"
 
         _args_schema = cls._args_schema
+        _args_schema.identity = AAZObjectArg(
+            options=["--identity"],
+            arg_group="Cluster",
+            help="Identity of Cluster resource",
+        )
```

Comment:
Could we remove these parameters which have been replaced by handwritten parameters?

Reply:
`args_schema.identity._registered = False` 
the parameter has been hidden, it is still used elsewhere,so it cannot be removed